quattor/types/site.pan

```diff
@@ -0,0 +1,9 @@
+declaration template quattor/types/site;
+
+@documentation{
+    site specific system structure entries
+}
+type structure_system_site = {
```

Comment:
Any example of what might go in this empty type?

Reply:
```
declaration template quattor/types/site;

include 'licenses/schema';

@documentation{
    UGent specific: Basic information needed for vsc-manage interaction.
}
type rundeck_info = { 
    @{Who governs an opennebula cluster being one, ...}
    "onemaster" ? string
};

type structure_system_ugent = { 
    "rundeck" ? rundeck_info
};

type structure_system_site = { 
    "quattorid"     : string = "-1"
    "licenses"      ? structure_license[]
    "ugent"         ? structure_system_ugent
};
```

Reply:
Ok thanks, `licenses` might be worth moving to an upstream schema though!